app/editor/src/components/form/formpage/FormPage.tsx

```diff
@@ -7,6 +7,8 @@ export interface IFormPageProps extends InputHTMLAttributes<HTMLDivElement> {
   minWidth?: string;
   /** bypass the default max width */
   maxWidth?: string;
+  /** remove padding from the FormPage */
+  noPadding?: boolean;
```

Comment:
Every time we add component properties for UI layout we're doing it wrong.  CSS specificity should be applied to override default behaviour as much as possible.

Reply:
Just need some clarification on the above @Fosol

Reply:
When we add a property in the component to control layout like this it makes it impossible for theme overrides based on css.  A component should have a standard look and feel.  If the implementation wants a different look, then they should just be able to add css to their solution and it will override the default.  You can do this through class name.  We can have a conversation about it today when you have time.

Reply:
Gotcha! Makes sense now